plugin/github_link_opener.vim

```diff
@@ -12,9 +12,12 @@ function! s:OpenGitHubLink()
   let chars = '[-a-zA-Z0-9\.]\+'
   let path = matchstr(word, chars . '/' . chars)
   let has_more_than_one_slash = word =~# '/.\+/'
+  let line_has_js_package = getline(".") =~# '\<\(require\|import\)\>'
   if &ft ==# 'go' && has_more_than_one_slash
     let url = matchstr(word, chars . '/' . chars . '/' . chars)
     call s:OpenWithNetrw("https://" . url)
+  elseif &ft ==# 'javascript' && line_has_js_package || &ft ==# 'json'
```

Comment:
I agree about the json issues - this was a half-baked idea. I'd expect most usage would come from reading js-like files anyway so dropping it doesn't seem like much of a loss.

I also agree that covering Typescript and React makes sense. I'm not fully steeped in modern javascript land yet so thanks for pointing these out.

I'll make the changes.

Reply:
I just realized that imports like
`import { filter } from 'rxjs/operators';` where `<cword>` from cursor on `operators` would pass `operators` rather than `rxjs` leading to unwanted behavior. I think we want to only use the first word before a slash. I'll try using a regex capture to get this working.